packages/mongo-menu/src/MongoMenu.tsx

```diff
@@ -255,9 +261,10 @@ function MongoMenu({
         <FocusableMenuItem>
           <Button
             size="small"
-            href={accountURL}
+            href={accountURL || undefined}
```

Comment:
(by explicitly setting this as undefined, `href` never gets provided to `Button` and we avoid a little console noise)